server/implementation/src/main/java/io/smallrye/graphql/scalar/GraphQLScalarTypes.java

```diff
@@ -97,6 +100,11 @@ public static void registerCustomScalar(
 
         mapType(new VoidScalar()); // Void
 
+        // Federation
+        mapType(new FieldSetScalar());
+        mapType(new PolicyScalar());
+        mapType(new ScopeScalar());
+
```

Comment:
Same reason for removing `Config.get().isFederationEnabled()` as in `io.smallrye.graphql.schema.model.Scalars`.